api-reference/50 Common/Object Structures/positionConfig/my/my.md

```diff
@@ -5,26 +5,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-The position of the widget to align against the target element.
+Specifies a side or corner of the overlay element that should be aligned with the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of).
```

Comment:
Can we say:
Specifies the overlay element's side or corner to align with a [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of).